sdk/eventgrid/eventgrid/CHANGELOG.md

```diff
@@ -1,14 +1,18 @@
 # Release History
 
-## 5.9.1 (Unreleased)
+## 5.10.0 (2025-01-21)
 
 ### Features Added
 
-### Breaking Changes
+- Added `internetMessageId` property to `AcsEmailDeliveryReportReceivedEventData`.
+- Added `recipientMailServerHostName` property to `AcsEmailDeliveryReportStatusDetails`.
+- For `AcsSmsReceivedEventData`:
+  - Added `segmentCount` property
 
-### Bugs Fixed
+### Bug fix
```

Comment:
I think the tooling might strictly check the header name here (it's not very smart) so you might have to leave it as it was originally.

Reply:
Done